parsimonious/expressions.py

```diff
@@ -151,10 +161,32 @@ def _uncached_match(self, text, pos, cache, error):
         if text.startswith(self.literal, pos):
             return Node(self.name, text, pos, pos + len(self.literal))
 
+    def _uncached_match_list(self, token_list, pos, cache, error):
```

Comment:
Thanks for taking the time to reply, @bloff! It sounds like I'm on the right track. To be clear, you needed to parse lists of tokens—[TOK1, TOK2, TOK3]—not hybrid lists with tokens and text—[TOK1, "some string", TOK2, TOK3]—correct?